easybuild/easyconfigs/c/CellRanger-ATAC/CellRanger-ATAC-2.1.0.eb

```diff
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Victor E. Goitea
+# Email: victorg@bmb.sdu.dk
+
+easyblock = 'Tarball'
+
+name = 'CellRanger-ATAC'
+version = '2.1.0'
+
+homepage = "https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/what-is-cell-ranger-atac"
+description = """Cell Ranger ATAC is a set of analysis pipelines that process 
+Chromium Single Cell ATAC data."""
+
+toolchain = SYSTEM
+
+# Download manually from https://support.10xgenomics.com/single-cell-atac/software/downloads/latest
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['3f44ba66ae789c2ab03f9c380a6718ca']
+
+keepsymlinks = True
+
+sanity_check_paths = {
+    'files': ['bin/cellranger-atac', 'cellranger-atac'],
+    'dirs': ['bin/atac', 'lib'],
+}
+
+sanity_check_commands = ["cellranger-atac testrun --id=tiny"]
+
+moduleclass = 'bio'
```

Comment:
Please also remove this newline - the unit tests are complaining.

Reply:
@verdurin I am not sure if I understand correctly which is the conflicting line, line 27 or 29? Is not 'bio' one of the defaults module classes?

Reply:
Thanks.